machineLearning/src/index.js

```diff
@@ -0,0 +1,56 @@
+export class Model {
+    constructor(image) {
+        this.image = image;
+    }
+
+    modeling() {
+        const modelPromise = cocoSsd.load().then((model) => {
+            const getValues = model.detect(this.image);
+            return getValues;
+        });
+        return modelPromise;
+    }
+
+    logging() {
+        const model = this.modeling();
+        model.then((logging) => {
+            console.log(logging);
+        });
+    }
+    selectData(objectSeacrched, percentage) {
+        let number = 0;
+        const model = this.modeling();
+        model.then((listObjects) => {
+            listObjects.forEach((element) => {
+                if (
+                    element.class === objectSeacrched &&
+                    element.score >= percentage
+                ) {
+                    number += 1;
+                }
+            });
+            console.log(
+                "Se han encontrado",
+                number,
+                "objetos tipo",
+                objectSeacrched,
+                "al",
+                percentage * 100,
+                "%"
+            );
+            const dataSelected = {
+                model: "cocoSsd",
+                objectS: objectSeacrched,
+                percetn: percentage,
+                quantity: number // file name and time
```

Comment:
I have a doubt in here, can you explain to me what does 'file name and time' means? Please

Reply:
This is the JSON object that the Machine Learning End point needs this object to send it

Reply:
Thanks Sergio, but I was talking about the comment. **_"quantity: number // file name and time"_** does the _**quantity**_ has those values on it?